forms-flow-theme/scss/_forms.scss

```diff
@@ -112,6 +112,56 @@ select option:hover {
   flex-direction: row;
 }
 
+.search-box {
+  .form-control-with-icon {
+    position: relative;
+    width: 25%;
+    
+
+    .search-box-input {
+      border-radius: 16px;
+      border: 1px solid var(--ff-primary);
+      background:  var(--ff-white);
+      color: var(--ff-gray-500);
+      font-size: 14px;
```

Comment:
Please use --font-size-xs instead of fontssize 14px.

Reply:
Updated

Reply:
can use --radius-0100 instead of px in border-radius